src/handlers/comment/handlers/query.ts

```diff
@@ -19,9 +19,9 @@ export const query = async (body: string) => {
     return `Skipping '/query' because of no issue instance`;
   }
 
-  const regex = /^\/query\s@(\w+)$/;
+  const regex = /^\/query\s+(@\w+)\s?$/;
```

Comment:
```suggestion
  const regex = /^\/query\s+@([\w-]+)\s*$/;
```

if you put `@` out of the capture group you don't need substring, username can contain `-` and we can allow for zero or multiple whitespace chars at the end

Reply:
We can simplify everything by just taking the comment corpus and trimming it `String.trim();` when it enters this function?